src/claFileRepository.ts

```diff
@@ -35,7 +35,7 @@ export class ClaFileRepository {
     private async getOrCreateClaFile(): Promise<[ClaFile, string]> {
         core.debug("Getting CLA file from source repository.");
         try {
-            const fileResult = await this.settings.octokitRemote.repos.getContents({
+            const fileResult = await this.settings.octokitRemote.repos.getContent({
```

Comment:
This function is mocked in the tests, we'll need to update the place where it's mocked to reflect this name change as well.

I tried running tests locally and noticed they were failing, but updating the mock was enough to get them passing again